Warn about potential crash in "Getting Started" (#1479)
doublesymmetry/react-native-track-player: docs/API.md

# Getting Started

## Starting off

> First of all, you need to register a [playback service](#playback-service) right after registering the main component of your app (typically in your `index.js` file at the root of your project):

Reviewer: ```suggestion
First you need to register a [playback service](#playback-service) right after registering the main component of your app (typically in your `index.js` file at the root of your project):
```

}
```

> Then you need to set up the player. This usually takes less than a second:

Reviewer: ```suggestion
Then, you need to set up the player. This usually takes less than a second:
```

```typescript
import TrackPlayer from 'react-native-track-player';

await TrackPlayer.setupPlayer()
// The player is ready to be used
```

> Make sure the setup method has completed before interacting with any other methods on `TrackPlayer` in order to avoid instability.

Reviewer: ```suggestion
Make sure the setup method has completed before interacting with any other functions in `TrackPlayer` in order to avoid instability.
```
